examples/tf_cloudbuild_workspace_simple/files/backend.tf

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+terraform {
+  backend "gcs" {
+    bucket = "tf-state-prod"
```

Comment:
do we need instructions to replace this fixed value ,`tf-state-prod`, with the bucket created in the seed project?

Reply:
For most use cases I am expecting this module to be used with inline config where we use partial backend config to override this. In SFB use a similar approach + subst with our custom build config that we ask users to copy over?

https://github.com/terraform-google-modules/terraform-google-bootstrap/blob/e83d8ea8e60aa95e3ba53c18c8fc4bae22ea2198/modules/tf_cloudbuild_workspace/cb.tf#L37